tests/test_read.py

```diff
@@ -0,0 +1,39 @@
+import pytest
```

Comment:
Looks like you're still missing tests for other file formats (pdf, txt, html). You'll want to get those in here too.

Reply:
Tests for read_pdf & read_txt implemented. read_html and its tests will be implemented in the next cycle.